apps/api/src/app/preferences/preferences.module.ts

```diff
@@ -1,10 +1,10 @@
 import { MiddlewareConsumer, Module, NestModule } from '@nestjs/common';
-import { GetPreferences, UpsertPreferences } from '@novu/application-generic';
+import { getFeatureFlag, GetPreferences, UpsertPreferences } from '@novu/application-generic';
 import { PreferencesRepository } from '@novu/dal';
 import { SharedModule } from '../shared/shared.module';
 import { PreferencesController } from './preferences.controller';
 
-const PROVIDERS = [PreferencesRepository, UpsertPreferences, GetPreferences];
+const PROVIDERS = [PreferencesRepository, UpsertPreferences, GetPreferences, getFeatureFlag];
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Was this accidentally omitted @davidsoderberg? I saw this in other modules that had feature flags‏

Reply:
It should not be needed as the SharedModule exports it and this module imports that module.